content/blogs/gui-setup-part3-first-testing-problems.md

```diff
@@ -0,0 +1,267 @@
+---
+title:      "1st-stage GUI setup - December 2023: First tests"
```

Comment:
It would be nice to have (Part 1, 2, 3, etc.) in the article title.

Reply:
Done here, and also for part 2 (PR #124) . I will apply this to the blog 1 too later.